protocol/tcp.go

```diff
@@ -77,3 +77,20 @@ func (t *TCP) UnmarshalBinary(data []byte) error {
 	return nil
 
 }
+
+// SplitData returns the TCP options and application layer message of the TCP packet.
+// It returns an error if HdrLen is invalid.
+func (t *TCP) SplitData() (options []byte, message []byte, err error) {
+	if t.HdrLen < 5 {
+		return nil, nil, errors.New("a TCP header must be at least 5 words")
```

Comment:
I think the coment should be "a TCP header must be at least 20 bytes" ?

Reply:
I've changed the comment